test/integration/test_groundlight.py

```diff
@@ -336,3 +336,44 @@ def test_submit_numpy_image(gl: Groundlight, detector: Detector):
     assert str(_image_query)
     assert isinstance(_image_query, ImageQuery)
     assert is_valid_display_result(_image_query.result)
+
+
+@pytest.mark.skipif(MISSING_NUMPY or MISSING_PIL, reason="Needs numpy and pillow")  # type: ignore
+def test_detector_improvement(gl: Groundlight, detector: Detector):
+    # test that we get confidence improvement after sending images in
+    # Pass two of each type of image in
+    import time
+
+    from PIL import Image
+
+    def submit_noisy_image(image):
+        np_noise = np.random.normal(loc=0, scale=50, size=image.shape)
+        noisy_img = np.clip(image + np_noise, 0, 255).astype(np.uint8)
+        img_query = gl.submit_image_query(detector=detector.id, image=noisy_img)
```

Comment:
```suggestion
        img_query = gl.submit_image_query(detector=detector.id, image=noisy_img, wait=0)
```

Let's set the `wait` here because otherwise the system will wait for human review.